src/main/java/org/benetech/servicenet/web/rest/SystemAccountResource.java

```diff
@@ -30,6 +32,7 @@
  */
 @RestController
 @RequestMapping("/api")
+@PreAuthorize("hasRole(\"" + AuthoritiesConstants.ADMIN + "\")")
```

Comment:
Will the USER be able to get info about its own system account?

Reply:
not via REST api